src/main/groovy/org/labkey/gradle/task/DeployApp.groovy

```diff
@@ -47,25 +44,12 @@ class DeployApp extends DeployAppBase
     @TaskAction
     void action()
     {
-        deployWebappDir()
         deployModules()
         deployPipelineJars()
         deployNlpEngine(deployBinDir)
         deployPlatformBinaries(deployBinDir)
     }
 
-
-    private void deployWebappDir()
-    {
-        ant.copy(
-                todir: deployWebappDir,
-                preserveLastModified: true
-        )
```

Comment:
We still need a `labkeywebapp` directory in our deployments. Will this directory get created in some other way?

Reply:
`/deploy/labkeyWebapp` is still getting created locally. I think this task was just copying the `/staging/webapp` files, of which there are now none.